client/landing/stepper/declarative-flow/write.ts

```diff
@@ -0,0 +1,137 @@
+import { useLocale } from '@automattic/i18n-utils';
+import { useFlowProgress, WRITE_FLOW } from '@automattic/onboarding';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { translate } from 'i18n-calypso';
+import { useEffect } from 'react';
+import { recordFullStoryEvent } from 'calypso/lib/analytics/fullstory';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import wpcom from 'calypso/lib/wp';
+import { useSiteSlug } from '../hooks/use-site-slug';
+import { USER_STORE, ONBOARD_STORE } from '../stores';
+import { recordSubmitStep } from './internals/analytics/record-submit-step';
+import LaunchPad from './internals/steps-repository/launchpad';
+import {
+	AssertConditionResult,
+	AssertConditionState,
+	Flow,
+	ProvidedDependencies,
+} from './internals/types';
+
+const write: Flow = {
+	name: WRITE_FLOW,
+	get title() {
+		return translate( 'Write' );
+	},
+	useSteps() {
+		const { resetOnboardStore } = useDispatch( ONBOARD_STORE );
+
+		useEffect( () => {
+			resetOnboardStore();
```

Comment:
@jeyip - Wondering if we still want to resetOnboardingStore() for these flows, given that users will be arriving at these flows fairly far into the onboarding process? That question will apply here, but for most other flows we define for general onboarding too.

Reply:
Makes sense -- still figuring out what to remove, but I'll update this